extensions/amp-3q-player/0.1/amp-3q-player.js

```diff
@@ -200,6 +203,15 @@ class Amp3QPlayer extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Forces the Layout to 'container' and sets a new height
+   * @private
+   * @param {number} newHeight
+   */
+  resize_(newHeight) {
+    this.attemptChangeHeight(newHeight);
```

Comment:
You might want to `catch` since this can trigger an error when rejected.
```suggestion
    this.attemptChangeHeight(newHeight).catch(() => {});
```